packages/http-canister-client/src/http-interface/http-interface.did

```diff
@@ -1,47 +0,0 @@
-// adapted from https://internetcomputer.org/docs/current/references/ic-interface-spec/#http-gateway-interface
```

Comment:
This file was renamed and formatted using the Motoko VSCode plugin, no meaningful changes.